.travis.yml

```diff
@@ -33,15 +33,22 @@ matrix:
 #            - BREW_INSTALL=python3
 
 install:
-  - pip install tox
+  - pip install flake8 tox
 #  - |
 #    if [[ $TRAVIS_OS_NAME == 'osx' ]]; then
 #      if [[ -n "$BREW_INSTALL" ]]; then
 #        brew update
 #        brew install "$BREW_INSTALL"
 #      fi
 #    fi
-#    pip install tox
+
+before_script:
+  # stop the build if there are Python syntax errors or undefined names
+  # exit-zero treats all errors as warnings.  The GitHub editor is 127 chars wide
+  if [[ $TOXENV == py37 ]]; then
+    flake8 . --count --exclude=./examples,./tests --select=E901,E999,F821,F822,F823 --show-source --statistics ;
```

Comment:
If we go the route of excluding specified directories instead of just including the **cmd2** directory, then for performance reasons when running locally, it is very important that we include "**./.***" so it doesn't try to parse directories such as:
* .cache
* .git
* .idea
* .pytest_cache
* .tox
* .vscode

I'm fine with this blacklisting approach, but the performance on my dev machine without excluding "**./.***" is orders of magnitude worse than running flake8 without any exclusions or running flake8 on  the cmd2 directory.

Reply:
On my Mac, the timing is as follows:
* 100 secs for **time flake8 --exclude=./examples,./tests**
*     2 secs for **time flake8 --exclude=./.*./examples,./tests**
*     2 secs for **time flake8 .**

Reply:
Put a . in the first one.